test/tests/functional/pbs_job_dependency.py

```diff
@@ -57,27 +57,26 @@ def setUp(self):
         TestFunctional.setUp(self)
         attr = {ATTR_RESC_TYPE: 'string', ATTR_RESC_FLAG: 'h'}
         self.server.manager(MGR_CMD_CREATE, RSC, attr, id='NewRes')
-        rv = self.scheduler.add_resource('NewRes', apply=True)
-        self.assertTrue(rv)
-        a = {ATTR_rescavail + '.ncpus': 0}
-        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
+        self.scheduler.add_resource('NewRes', apply=True)
+        self.create_vnodes()
+
+    def cust_attr(self, name, totnodes, numnode, attrib):
+        res_str = "ver" + str(numnode)
+        attr = {'resources_available.NewRes': res_str}
```

Comment:
but usenatvnode is False by default. Just to be sure, I'll specifically call out usenatvnode as False.

Reply:
I wasn't asking you to change anything.  I was just warning you for the future.  I ran into. this issue when I tested node buckets.

Reply:
That's good advice! I just thought that if someone changes the default in the library then it will break the test. So it is probably better to explicitly specify the option.